pkg/pillar/types/zedroutertypes.go

```diff
@@ -325,6 +372,23 @@ func (portConfig DevicePortConfig) WasDPCWorking() bool {
 	return false
 }
 
+// UpdatePortStatusFromIntfStatusMap - Set Port Errors for ports in portConfig to
```

Comment:
s/Errors/status/

Reply:
Cleaned up the comment in PR: https://github.com/lf-edge/eve/pull/890